src/components/articles/AllArticlesCard.jsx

```diff
@@ -0,0 +1,17 @@
+import "./AllArticles.css";
+
+export default function AllArticlesCard({ article }) {
+  return (
+    <li className="all-articles-card">
+      <h2 id="all-articles-title">{article.title}</h2>
+      <h3 id="all-articles-topic">{article.topic}</h3>
+      <p id="all-articles-author">{article.author}</p>
+      <img id="all-articles-img" src={article.article_img_url} />
+      <div id="all-articles-votes-comments">
+        <div>♥️ {article.votes}</div>
+        <div>💬 {article.comment_count}</div>
+      </div>
+      <time id="all-articles-date">{article.created_at}</time>
```

Comment:
Love the use of semantic component here 
This component is looking great, I'd say the only thing letting it down is how unintelligle the date format is
Consider using a lightweight library like https://day.js.org/ to make the date a bit more readable 📆